core/controllers/ingest.py

```diff
@@ -50,7 +52,9 @@ def ingest(body: dict, excel_file: FileStorage) -> Response:
     if reporting_round in [1, 2]:
         populate_db_historical_data(workbook, INGEST_MAPPINGS)
     else:
-        populate_db(workbook, INGEST_MAPPINGS)
+        with_transaction(
+            populate_db, 3, 0.4, error_type=exc.IntegrityError, workbook=workbook, mappings=INGEST_MAPPINGS
```

Comment:
think it will be clearer to use keyword arguments for the max retries and sleep duration here